docker-hub/repos/index.md

```diff
@@ -72,7 +78,7 @@ To avoid losing your private images, you can pull your private images from your
 
 1. Navigate to [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} and select **Organizations**.
 2. Select the applicable organization and verify that your user account is a member of the organization.
-3. Sign in to [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} using your original Docker ID, and pull your images from the initial namespace.
+3. Sign into [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} using your original Docker ID, and pull your images from the initial namespace.
```

Comment:
Should this stay as `Sign in to`? This always trips me up, so I'm not sure.

Reply:
Same =) Thank you! Updated for sign in to